examples/text-generation/run_lm_eval.py

```diff
@@ -110,14 +110,13 @@ def __init__(self, tokenizer, model, args, options):
             "qwen2",
             "gptj",
             "starcoder2",
-            "gemma",
```

Comment:
Can you check who added this gemma model for reuse_cache to be enabled here, and ask him/her to review your change?

Reply:
Sure @jiminha. @dsmertin  Please review this PR.

Reply:
@skavulya  any update after last comment?

Reply:
I updated the Gemma causal model to support `attn_softmax_bf16`. The reuse cache and flash attention were already supported